NosTests/GiftWrapperTests.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+
+final class GiftWrapperTests: XCTestCase {
+    func testDirectMessageWrapper() throws {
```

Comment:
Maybe this could be in a new class called `DirectMessageWrapperTests`? Not 100% sure about the test below since it uses both DirectMessageWrapper and GiftWrapper.

Reply:
Implemented in https://github.com/planetary-social/nos/pull/1048/commits/b07ed7095ef1536f7e98646c5c69d25166dbfecd